### PR TITLE
add param to values section

### PR DESCRIPTION
**Pull request by JacobGros**
# Description
Adds documentation for fsgroup in pflex 
Corresponds to: https://github.com/dell/csi-powerflex/pull/86 

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/256 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [x] Have you added high-resolution images?

### CODE REVIEW

**Review by abhi16394 — DISMISSED** [review]

**Review by jooseppi-luna — COMMENTED** [review]
Currently "Supported" is capitalized in the middle of a sentence, perhaps change to this:

Defines which FS Group policy mode to be used, supported modes are `None`, `File` and `ReadWriteOnceWithFSType`

Edit: also adjusting backticks so they don't include the "and".

**Review by bharathsreekanth — APPROVED** [review]

**Review by shanmydell — CHANGES_REQUESTED** [review]
Add the release notes update

**Comment by shanmydell** [comment]
@JacobGros : Could we get this PR updated?

**Comment by shanmydell** [comment]
@JacobGros : Operator changes are missing, apologies for missing out in earlier comment

**Comment by JacobGros** [comment]
> @JacobGros : Operator changes are missing, apologies for missing out in earlier comment

Currently operator is not supported- will update this PR when support is added

File: content/docs/csidriver/installation/helm/powerflex.md [hunk 1]
```diff
@@ -206,6 +206,7 @@ kubectl create -f deploy/kubernetes/snapshot-controller
 | logFormat | CSI driver log format. Allowed values: "TEXT" or "JSON". | Yes | "TEXT" |
 | kubeletConfigDir | kubelet config directory path. Ensure that the config.yaml file is present at this path. | Yes | /var/lib/kubelet |
 | defaultFsType | Used to set the default FS type which will be used for mount volumes if FsType is not specified in the storage class. Allowed values: ext4, xfs. | Yes | ext4 |
+| fsGroupPolicy | Defines which FS Group policy mode to be used. Supported modes are`None, File, and ReadWriteOnceWithFSType.` | No | "ReadWriteOnceWithFSType" |
```
Comment (randeepsharma) [inline content/docs/csidriver/installation/helm/powerflex.md:209]: In the values file the default is File.
https://github.com/dell/csi-powerflex/blob/main/helm/csi-vxflexos/values.yaml#L80

**Pull request merged**